task24/js/main.js

```diff
@@ -0,0 +1,54 @@
+import "../css/style.css";
+
+const modalWrapper = document.getElementById("js-modalContents");
+const modal = document.getElementById("js-modal");
+const modalContents = document.getElementById('js-modalContents');
+const modalPlace = document.getElementById("js-modalPlace");
+const closeModal = document.getElementById("js-modalClose");
+const closeModalText = document.getElementById("js-modalCloseText");
+const checkBox = document.getElementById("js-checkBox");
+const login = document.getElementById("js-login");
+
+const createModal = () => {
+    modalWrapper.classList.remove('hidden');
+    modalWrapper.classList.add("fixed", "inset-0", "flex",  "justify-center", "items-center", "z-50", "bg-gray-700", "bg-opacity-95")
+    modal.classList.add("max-w-screen-sm", "h-96", "p-4", "bg-white", "overflow-y-auto", "rounded-lg")
+    closeModal.classList.add("flex", "justify-center", "items-center", "p-6", "hover-item")
+    closeModalText.classList.add("hover-text")
+}
```

Comment:
IMO
Since modal already exists in HTML, why not just put the class in the HTML side?

IMO
Since I didn't CREATE it, I thought it would be better to have a different function name.

Reply:
As you say. Thanks for your spot on comment! I have corrected it. I also learned a lot from your comment about naming functions.
[0970040](https://github.com/seeeen044/JavaScript_handson/pull/26/commits/0970040fe922c7d4106e9cdce415a70b91da6de9)